db/migrations/20230512113441_sensei_migration.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class SenseiMigration extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up(): void
+    {
+        // sensei table
+        $this->execute("
+            CREATE TABLE `sensei` (
+                `sensei_id` int(11) NOT NULL,
```

Comment:
To make this clear, could we change this field to `sensei_user_id` or just `user_id`? It's a little weird reading through the rest of the code and trying to understand if there's a case where `sensei_id` would not match the user's ID

Reply:
For context:
When a user has `sensei_id` == `user_id` it indicates they are the sensei.
When a user has `sensei_id` != `user_id` it indicates they are a student.
If `sensei_id` == NULL || 0 they are neither.

The sensei table uses `sensei_id` as the primary key to be clear it corresponds to the `sensei_id` in the user table.

I'm fine changing it to `sensei_user_id`, just want to keep it consistent between the user and sensei table. 
Thoughts?